src/bridge/cockpitpackages.c

```diff
@@ -1073,15 +1072,19 @@ package_content (CockpitPackages *packages,
             }
         }
 
+      /* If the response is language specific, don't cache the file. The only known case is "po.js",
+       * caching breaks changing the language in the browser. */
```

Comment:
This comment could also benefit from a bit more exposition on the topic.  At least a link to the relevant issue would be nice.

Reply:
I changed it, WDYT?